appimage/build_appimage.sh

```diff
@@ -168,24 +177,26 @@ build_frontend() {
     popd || handle_error
 }
 
-build_appimage() {
-    log_message "Building AppImage"
-    appimage-builder --recipe monkey_island_builder.yml --log DEBUG --skip-appimage
+add_monkey_icon() {
+	unlink "$APPDIR"/python.png
+	mkdir -p "$APPDIR"/usr/share/icons
+	cp "$REPO_MONKEY_SRC"/monkey_island/cc/ui/src/images/monkey-icon.svg "$APPDIR"/usr/share/icons/infection-monkey.svg
+	ln -s "$APPDIR"/usr/share/icons/infection-monkey.svg "$APPDIR"/infection-monkey.svg
+}
 
-    # There is a bug or unwanted behavior in appimage-builder that causes issues
-    # if 32-bit binaries are present in the appimage. To work around this, we:
-    #   1. Build the AppDir with appimage-builder and skip building the appimage
-    #   2. Add the 32-bit binaries to the AppDir
-    #   3. Build the AppImage with appimage-builder from the already-built AppDir
-    #
-    # Note that appimage-builder replaces the interpreter on the monkey agent binaries
-    # when building the AppDir. This is unwanted as the monkey agents may execute in
-    # environments where the AppImage isn't loaded.
-    #
-    # See https://github.com/AppImageCrafters/appimage-builder/issues/93 for more info.
-    download_monkey_agent_binaries
+add_desktop_file() {
```

Comment:
And if OS is running without a GUI? User will get a file in "$APPDIR"/infection-monkey.desktop which has no effect, right?

Reply:
`$APPDIR` is the directory that contains the [AppDir](https://docs.appimage.org/reference/appdir.html), which is the source of the appimage.  The AppImage is delivered as a single file. The .desktop file is part of the appdir spec. It should have no effect if the OS has no GUI.